package.json

```diff
@@ -57,7 +57,7 @@
         "zone.js": "^0.10.3"
```

Comment:
Avoid making changes to the `package.json`. If it is absolutely necessary let me know and I'll run it by Keyvan.